frontend/src/app/components/wp-single-view-tabs/additional-tab/tab.ts

```diff
@@ -0,0 +1,10 @@
+import { Type } from '@angular/core';
+import { TabComponent } from './tab.component';
+
+export class Tab {
+  constructor(
+    public component: Type<TabComponent>,
+    public displayName: String,
+    public identifier: String,
+  ) {}
+}
```

Comment:
is this an OK place for a new ts class or is there a more central place where we place these kind of meta-classes?